ISSUE_TRIAGE.rst

```diff
@@ -0,0 +1,92 @@
+
+This document is a hands-on manual for doing issue and pull request triage for `MLflow issues 
+on GitHub <https://github.com/mlflow/mlflow/issues>`_ . 
+The purpose of triage is to speed up issue management and get community members faster responses. 
+
+Issue and pull request triage has three steps:
+
+- assign one or more process labels (e.g. ``needs design`` or ``help wanted``),
```

Comment:
Nit: There's a dangling comma at the end of this line

Reply:
That was intended. Do
- a, 
- b, and
- c.